sdk/core/core/test/cmocka/az_test_definitions.h

```diff
@@ -1,63 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#ifndef _az_TEST_DEFINITIONS_H
```

Comment:
Don't we still need these?

Reply:
not really, We are just going to import it once from main.
Removing this actually help to break if it is added again some how